lib/convection/control/cloud.rb

```diff
@@ -114,6 +114,18 @@ def diff(to_stack, options = {}, &block)
           return
         end
 
+        errors = []
+        stacks.each_value do |stack|
+          if stack.error?
+            errors << stack.errors.collect { |x| x.exception.message }
+          end
+        end
+        errors = errors.uniq.flatten!
+        unless errors.empty?
+          block.call(Model::Event.new(:error, "Error(s) during stack diff #{errors.join(', ')}", :error), errors) if block
+          return
+        end
```

Comment:
Should we change the converge behavior to match the diff behavior?  If we report errors on all stacks when only asking to diff a specific stack, we should report errors on all stacks when asking to converge a specific stack.  Any reason for the inconsistency.

Let's open a bug to fix the exit status.  CLI tools should return non-zero exit codes on failures.